docs/how_to_add_packages.md

```diff
@@ -113,34 +113,27 @@ versions:
 This file lists **all the sources that are needed to build the package**: source code, license files,... any file that will be used by the recipe
 should be listed here. The file is organized into two sections, `sources` and `patches`, each one of them contains the files that are required
 for each version of the library. All the files that are downloaded from the internet should include a checksum, so we can validate that
-they are not changed:
+they are not changed.
 
-```yml
-sources:
-  "1.1.0":
-    url: "https://www.url.org/source/mylib-1.0.0.tar.gz"
-    sha256: "8c48baf3babe0d505d16cfc0cf272589c66d3624264098213db0fb00034728e9"
-  "1.1.1":
-    url: "https://www.url.org/source/mylib-1.0.1.tar.gz"
-    sha256: "15b6393c20030aab02c8e2fe0243cb1d1d18062f6c095d67bca91871dc7f324a"
-patches:
-  "1.1.1":
-    - patch_file: "patches/1.1.1-001-simpler-cmakelists.patch"
-      base_path: "source_subfolder"
-```
+A detailed breakdown of all the fields can be found in [conandata_yml_format.md](conandata_yml_format.md). We strongly encourage adding the [patch feilds](conandata_yml_format.md#patches-fields) to help track where patches come from and what issue they solve.
 
 Inside the `conanfile.py` recipe, this data is available in a `self.conan_data` attribute that can be used as follows:
 
 ```py
 def source(self):
-     tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
+    tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
```

Comment:
Use tools from v1/v2 compatible layer: `conan.tools.files.get(self,...)`

Reply:
Can I do this in a separate PR? 🙏 

A TON of docs needs v2 migration attention and I dont want to mix more changes

Reply:
```suggestion
    files.get(self, **self.conan_data["sources"][self.version], destination=self.source_folder, strip_root=True)
```